app/Console/Commands/Init.php

```diff
@@ -32,40 +41,192 @@ public function handle(): void
+        $this->info('Project initialized successfully!');
 
-        if ($this->confirm('Do you want generate admin user?', true)) {
+        if ($this->confirm('Do you want to generate an admin user?', true)) {
             $this->createAdminUser($kebabName);
         }
+
+        if ($this->confirm('Do you want to generate a README file?', true)) {
+            $this->fillReadme();
+
+            if ($this->confirm('Do you need a `Resources & Contacts` part?', true)) {
+                $this->fillResourcesAndContacts();
+                $this->fillResources();
+                $this->fillContacts();
+            }
+
+            if ($this->confirm('Do you need a `Prerequisites` part?', true)) {
+                $this->fillPrerequisites();
+            }
+
+            if ($this->confirm('Do you need a `Getting Started` part?', true)) {
+                $this->fillGettingStarted();
+            }
+
+            if ($this->confirm('Do you need an `Environments` part?', true)) {
+                $this->fillEnvironments();
+            }
+
+            if ($this->confirm('Do you need a `Credentials and Access` part?', true)) {
+                $this->fillCredentialsAndAccess();
+            }
+
+            $this->saveReadme();
+
+            $this->info('README generated successfully!');
+
+            if ($this->emptyValuesList) {
+                $this->warn('Don`t forget to fill the following empty values:');
+
+                foreach ($this->emptyValuesList as $value) {
+                    $this->warn("- {$value}");
+                }
+            }
+        }
     }
 
     protected function createAdminUser($kebabName): void
     {
         $defaultPassword = substr(md5(uniqid()), 0, 8);
 
-        $this->publishMigration([
-            'name' => $this->ask('Please enter admin name', 'Admin'),
-            'email' => $this->ask('Please enter admin email', "admin@{$kebabName}.com"),
-            'password' => $this->ask('Please enter admin password', $defaultPassword),
+        $this->adminCredentials = [
+            'name' => $this->ask('Please enter an admin name', 'Admin'),
+            'email' => $this->ask('Please enter an admin email', "admin@{$kebabName}.com"),
+            'password' => $this->ask('Please enter an admin password', $defaultPassword),
             'role_id' => Role::ADMIN
-        ]);
+        ];
+
+        $this->publishMigration();
+    }
+
+    protected function fillReadme(): void
+    {
+        $appName = $this->argument('application-name');
+        $file = $this->loadReadmePart('README.md');
+
+        $this->setReadmeValue($file, 'project_name', $appName);
+
+        $this->readmeContent = $file;
+    }
+
+    protected function fillResourcesAndContacts(): void
+    {
+        $filePart = $this->loadReadmePart('RESOURCES_AND_CONTACTS.md');
+
+        $this->updateReadmeFile($filePart);
+    }
+
+    protected function fillResources(): void
+    {
+        $resources = [
```

Comment:
let's move it to the class const or protected field

Reply:
Done